Resources/ServerInfo/Guidebook/NewPlayer.xml

```diff
@@ -0,0 +1,12 @@
+<Document>
+# Welcome to Space Station 14!
+
+Welcome to the pre-alpha of Space Station 14! We hope you enjoy the game, and this entry will serve to guide you on how to learn to play. There's quite a bit to read, but SS14 is in itself a very large and in-depth game, and hopefully these guides help you enjoy it to it's fullest.
```

Comment:
```suggestion
Welcome to the Steam Playtest of Space Station 14! We hope you enjoy the game, and this entry will serve to guide you on how to learn to play. There's quite a bit to read, but SS14 is in itself a very large and in-depth game, and hopefully these guides help you enjoy it to it's fullest.
```
Or alternatively just `alpha`, but it's inaccurate to call SS14 "pre-alpha".

Reply:
wait but you can play on other platforms, no? cant you download the launcher directly from the website?

Reply:
just call it the playtest then